Course.php

```diff
@@ -293,7 +295,7 @@
 FROM `lab_report_submissions`
 Left JOIN users_table  on users_table.Student_ID=lab_report_submissions.Student_id
 left JOIN course_group_members_table on course_group_members_table.Course_Group_id=lab_report_submissions.Course_Group_id
-where Lab_Report_ID=$lab_repo_id and lab_report_submissions.Student_id='$student_id'"); 
+where Lab_Report_ID=$lab_repo_id and lab_report_submissions.Student_id='$student_id' or lab_report_submissions.Course_Group_id='$group_id'"); 
```

Comment:
Do we still need the following conditional test?

`Lab_Report_ID=$lab_repo_id and lab_report_submissions.Student_id='$student_id'`


**Hui**